code/datums/uplink/announcements.dm

```diff
@@ -1,6 +1,8 @@
 /****************
 * Announcements *
 *****************/
+var/cooldown = 0
+var/cooldownr = 0
```

Comment:
Not fixed. These are still here. Which is no bueno

```
/datum/uplink_item/abstract/announcements/fake_ion_storm/get_goods(var/obj/item/device/uplink/U, var/loc)
	var/static/cooldown
	if(cooldown != 1)
		ion_storm_announcement()
		cooldown = 1
		spawn(240)
			cooldown = 0
		return 1
	else
		loc << "<span class='danger'>This service is on cooldown! Try again in a bit!</span>"
		return 0
```

^ This is what I meant. And you need to delete the top ones from up there.